src/data/protocols/index.ts

```diff
@@ -1,2 +1,8 @@
-export * from './add-guardian-repository'
-export * from './encrypter'
+export * from './cryptography/hash-comparer'
+export * from './cryptography/hash-generator'
+export * from './cryptography/token-generator'
+export * from './cryptography/token-decoder'
```

Comment:
The files in the "cryptography" directory must be exported in an "index.ts" file inside the directory itself.